metaflow/metaflow_config.py

```diff
@@ -218,6 +218,8 @@
 
 RUNTIME_CARD_RENDER_INTERVAL = from_conf("RUNTIME_CARD_RENDER_INTERVAL", 60)
 
+CARD_WRITE_TO_METADATA = from_conf("CARD_WRITE_TO_METADATA", True)
```

Comment:
do we need to offer it as a top level config? we can start writing metadata always in the context of flows.

Reply:
@tuulos recommended that we keep the writing optional since it was not there earlier. Personally don't have a problem with removing it since the UI service might still use the same logic that retrieves stuff from the datastore.

Reply:
We discussed this in the weekly chat that it would be easy to keep things simple and consistent going forward.